src/engine/specification.md

```diff
@@ -197,25 +223,35 @@ This structure contains the attributes required to initiate a payload build proc
 #### Response
 
 * result: `object`
-    - `status`: `enum` - `"SUCCESS" | "SYNCING"`
+    - `status`: `enum` - `"VALID" | "INVALID" | "SYNCING" | "ACCEPTED"`
+    - `latestValidHash`: `DATA|null`, 32 Bytes - the hash of the most recent *valid* block in the branch defined by payload and its ancestors
+    - `validationError`: `String|null` - a message providing additional details on the validation error if the payload is deemed `INVALID`
     - `payloadId`: `DATA|null`, 8 Bytes - identifier of the payload build process or `null`
-* error: code and message set in case an exception happens while updating the forkchoice or initiating the payload build process.
+* error: code and message set in case an exception happens while validating payload, updating the forkchoice or initiating the payload build process.
```

Comment:
```suggestion
* error: code and message set in case an exception happens while the validating payload, updating the forkchoice or initiating the payload build process.
```